cms/static/cms/js/build.js

```diff
@@ -171,5 +171,4 @@
          */
         logLevel: 1
     };
-}())  // eslint-disable-line semi
-// A semicolon on the line above will break the requirejs optimizer
+}());
```

Comment:
Cool, that gives me more confidence that this will be an okay change to merge. Thanks!